libs/domain/picking/src/feature.ts

```diff
@@ -66,39 +48,33 @@ export interface PickingFeatureConfig extends PickingConfig {
   noDefaultRoutes?: boolean;
 }
 
-export class PickingFeature implements AppFeature {
+export class PickingFeature extends PickingApiFeature {
   providers: Provider[];
   components: ComponentsInfo;
 
   constructor(config?: PickingFeatureConfig) {
+    super();
     this.providers = this.getProviders(config);
     this.components = pickingComponents;
   }
 
-  protected getProviders(config?: PickingFeatureConfig): Provider[] {
+  protected override getProviders(config?: PickingFeatureConfig): Provider[] {
     return [
       ...provideLitRoutes(
         !config?.noDefaultRoutes ? { routes: defaultPickingRoutes } : undefined
       ),
       ...providePickingConfig(config),
       {
-        provide: JsonAPITransformerService,
-        useClass: DefaultJsonAPITransformerService,
-      },
-      ...warehouseUserAssignmentNormalizer,
-      ...warehouseUserAssignmentsNormalizer,
-      { provide: PickingListService, useClass: PickingListDefaultService },
-      { provide: PickingListAdapter, useClass: PickingListDefaultAdapter },
-      { provide: PickingHttpService, useClass: PickingHttpDefaultService },
-      { provide: PickingHeaderService, useClass: PickingHeaderDefaultService },
-      {
-        provide: WarehouseUserAssignmentsAdapter,
-        useClass: WarehouseUserAssignmentsDefaultAdapter,
+        provide: PickingHeaderService,
+        useClass: PickingHeaderDefaultService,
       },
+      //override SapiLocaleAdapter that is provided by siteFeature with default one
```

Comment:
Can you please add a TODO with the ticket number for it please?

Reply:
This hacky fix was implemented previously, I've just moved the service overriding to another place (previously it was in preset, if I'm not mistaken). Added link to the original ticket in the comment